plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineFlexWarFacet.java

```diff
@@ -61,6 +62,7 @@ public static void installAppEngineFacet(IFacetedProject facetedProject,
     FacetUtil facetUtil = new FacetUtil(facetedProject);
     if (installDependentFacets) {
       facetUtil.addJavaFacetToBatch(JavaFacet.VERSION_1_8);
+      facetUtil.addWebFacetToBatch(WebFacetUtils.WEB_31);
```

Comment:
Adding the WTP facet is the correct behavior.

No one called this method before, this doesn't break anything.